### DISCUSSION TITLE
padding=ft.Padding.only

### DISCUSSION THREAD

**Discussion by abcnow** [flet-dev/flet#2575] [General]
I have this code:

controls=[
                        ft.ResponsiveRow(
                            controls=[
                                ft.Container(
                                    col={
                                        "sm": 6,
                                        "md": 3,
                                    },
                                    border_radius=10,
                                    padding=ft.Padding.only(top=70, bottom=70),
                                    bgcolor=ft.colors.GREY,
                                ),

and it is giving me this error: type object: 'Padding' has no attribute 'only'... I searched for it on the docs but it is not clear how to solve this problem.. thanks in advance for your help!

**Comment by ndonkoHenri** [comment]
`ft.padding.only` - `padding` with small p.

**Reply by abcnow** [reply]
Thanks a lot man! @ndonkoHenri It worked :)
